src/fragments/fragments.ts

```diff
@@ -16,7 +16,7 @@ export const interpretFragments = (fragments: VerificationFragment[]): interpret
 
 export const errorMessageHandling = (fragments: VerificationFragment[]): string[] => {
   const { hashValid, issuedValid, identityValid } = interpretFragments(fragments);
-  const errors = [];
+  let errors = [];
```

Comment:
The changes seem to only use `errors.push`? Then maybe we can still use `const`?

Reply:
yea i was testing something out; forgot to change back to `const` will do so